tests/drivers/nrfx_integration_test/src/main.c

```diff
@@ -15,6 +15,22 @@
 
 NRFX_STATIC_ASSERT(true);
 
+#if defined(NRFX_GPIOTE0_CHANNELS_USED)
+#define NRFX_GPIOTE_CHANNELS_USED NRFX_GPIOTE0_CHANNELS_USED
+#elif defined(NRFX_GPIOTE1_CHANNELS_USED)
+#define NRFX_GPIOTE_CHANNELS_USED NRFX_GPIOTE1_CHANNELS_USED
+#elif defined(NRFX_GPIOTE20_CHANNELS_USED)
+#define NRFX_GPIOTE_CHANNELS_USED NRFX_GPIOTE20_CHANNELS_USED
+#elif defined(NRFX_GPIOTE30_CHANNELS_USED)
+#define NRFX_GPIOTE_CHANNELS_USED NRFX_GPIOTE30_CHANNELS_USED
+#elif defined(NRFX_GPIOTE130_CHANNELS_USED)
+#define NRFX_GPIOTE_CHANNELS_USED NRFX_GPIOTE130_CHANNELS_USED
+#elif defined(NRFX_GPIOTE131_CHANNELS_USED)
+#define NRFX_GPIOTE_CHANNELS_USED NRFX_GPIOTE131_CHANNELS_USED
+#else
+#define NRFX_GPIOTE_CHANNELS_USED 0
+#endif
```

Comment:
Instead of adding this stuff, I think we should just remove the following line (`NRFX_GPIOTE_CHANNELS_USED` no longer needs to be defined): https://github.com/nrfconnect/sdk-nrf/blob/35ebad3654c76569b68a6134508d88e5461739f1/tests/drivers/nrfx_integration_test/src/main.c#L71

Reply:
done